rust/Cargo.toml

```diff
@@ -56,9 +56,9 @@ async-stream = { version = "0.3.2", default-features = true, optional = true }
 # High-level writer
 parquet-format = "~2.6.1"
 
-arrow  = { git = "https://github.com/apache/arrow-rs", rev = "615d7830fdaf994ebd32ca1e349daf68b18c99b0" }
-parquet = {  git = "https://github.com/apache/arrow-rs", rev = "615d7830fdaf994ebd32ca1e349daf68b18c99b0" }
-datafusion = { git = "https://github.com/houqp/arrow-datafusion", rev = "af34cec956c8d67b2520a266e74683d7bdcb3099", optional = true }
+arrow = "6.1.0"
+parquet = "6.1.0"
+datafusion = { git = "https://github.com/apache/arrow-datafusion", rev = "7060961fd944efa48c7ed93677041f4d089b5da5", optional = true }
```

Comment:
@houqp the latest main uses arrow 6.1, but my poor knowledge of this crate blocks me of resolving the delta-datafusion module compilation. There's some significant api change for the `ParquetExec`

Reply:
yeah... I can help with the migration this weekend if it's not urgent. Otherwise, we can temporarily disable datafusion for now.

Reply:
sure, there's no rush whatsoever